src/gossip/helpers.zig

```diff
@@ -0,0 +1,53 @@
+const std = @import("std");
+const sig = @import("../sig.zig");
+
+const Pubkey = sig.core.Pubkey;
+const ContactInfo = sig.gossip.ContactInfo;
+const Logger = sig.trace.Logger;
+const Network = sig.cmd.config.Network;
```

Comment:
I lean towards putting this in `core`. Also the name `Cluster` is more accurate than `Network`. The entrypoints function could be defined in gossip though. We could also have a config folder outside cmd but I'd be careful about lumping too much in there.